tests/test_visitors/test_ast/test_decorators/test_new_style_decorators.py

```diff
@@ -18,6 +20,42 @@ class Some:
     def some(self): ...
 """
 
+if sys.version_info >= (3, 12):
+    wrong_decorators = [
```

Comment:
Create common lists of wrong and correct decorators. Then extend them with version specific ones.